src/userbehavior/userbehavior/usbing/usbing.py

```diff
@@ -137,7 +138,11 @@ def open_saved_exes(self):
 
     def mount_usb_device(self):
         logger.info("Mounting USB device")
-        self.usb_device.mount()
+        for i in range(self.max_mount_retries):
+            self.usb_device.mount()
+            if self.usb_device.is_mounted():
+                break
+            logger.info(f"Mounting attempt {i+1}/{self.max_mount_retries} failed.")
 
```

Comment:
IMHO this is not necessary. The usbing is triggered every few seconds anyway, so if something goes wrong we can simply cancel and retry on the next run.